projectile.el

```diff
@@ -2654,6 +2654,65 @@ test/impl/other files as below:
   :type 'function
   :package-version '(projectile . "1.0.0"))
 
+(defun projectile--cmake-version()
+  "Compute CMake version."
+  (let* ((string (shell-command-to-string "cmake --version"))
+         (match (string-match "^cmake version \\([0-9]*\\)\.\\([0-9]*\\)" string)))
+    (when match
+      (mapcar 'string-to-number
```

Comment:
Why don't you just use `version-to-list` here?

Reply:
Good suggestion, didn't know this existed.

Reply:
Fixed.